lightning/src/routing/scoring.rs

```diff
@@ -801,6 +801,13 @@ struct ChannelLiquidity<T: Time> {
 	max_liquidity_offset_history: HistoricalBucketRangeTracker,
 }
 
+impl<T: Time> ChannelLiquidity<T> {
+	/// The difference between the upper and lower liquidity bounds.
+	fn liquidity_offset_delta(&self) -> u64 {
+		self.max_liquidity_offset_msat - self.min_liquidity_offset_msat
```

Comment:
These are relative offsets (i.e., from the channel capacity / max htlc and zero, respectively). You want the absolute, decayed values instead. Otherwise, no knowledge (i.e., offsets of zero) would be preferred. See `min_liquidity_msat` and `max_liquidity_msat` instead, which also account for decays.

Please also add tests. :)

Reply:
Those are available for the directed version, I can put fake info to get those, otherwise not sure the best way to handle this

Reply:
I think just adding a private function using the canonical order is fine. It doesn't matter which way they are directed so long as they are the same direction when compared.